src/Microsoft.AspNetCore.SignalR.Core/DefaultHubLifetimeManager.cs

```diff
@@ -176,7 +195,7 @@ private InvocationMessage CreateInvocationMessage(string methodName, object[] ar
 
         public override Task InvokeUserAsync(string userId, string methodName, object[] args)
         {
-            return InvokeAllWhere(methodName, args, connection =>
+            return InvokeFirstWhere(methodName, args, connection =>
```

Comment:
I don't think we want this. You can have the same user connected from multiple locations and with this change only one would get the message, but all of them should get the message.

Reply:
Hmmm, good point. Didn't think of that.

Reply:
Should there be a better way to may User Identifies to connection ids though?